lib/converterbase.rb

```diff
@@ -91,7 +91,15 @@ def rstrip_all_lines(data)
   #
   def convert_numbers(data)
     # 小数点を・に
-    data.gsub!(/([\d０-９#{KANJI_NUM}]+?)[\.．]([\d０-９#{KANJI_NUM}]+?)/, "\\1・\\2")
+    data.gsub!(/([\d０-９#{KANJI_NUM}]+?)[\.．]([\d０-９#{KANJI_NUM}]+?)/) do |match|
+      integer = $1
+      decimal = $2
+      if integer[-1] =~ /\d/ && decimal[0] =~ /\d/ || integer[-1] =~ /[０-９]/ && decimal[0] =~ /[０-９]/ || integer[-1] =~ /[#{KANJI_NUM}]/ && decimal[0] =~ /[#{KANJI_NUM}]/
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [170/140] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->